### ISSUE TITLE
Super/Meta+L opens the Melange debugger by default instead of locking the workspace

### ISSUE THREAD

**Issue by ciphersimian** [linuxmint/cinnamon#10984]
```
$ cinnamon --version
Cinnamon 5.4.8
$ cat /etc/os-release 
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
BUILD_ID=rolling
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="32;1;24;144;200"
HOME_URL="https://manjaro.org/"
DOCUMENTATION_URL="https://wiki.manjaro.org/"
SUPPORT_URL="https://manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux
 ```

**Issue**
Issue #3741 needs to be revisited (Super+L opens the Melange debugger by default instead of locking the workspace).

**Steps to reproduce**
Press Super+L and observe as it opens the little used (by the general user population) Melange debugger.

**Expected behaviour**
Lock the workspace as it does in nearly every other graphical computing environment in existence.

**Other information**
In the original issue this was framed as Windows behavior, but that is patently false.  I've been a developer for over 25 years and within 5 feet of me I have systems running Manjaro, Debian 11, Ubuntu 22.04, Windows 10, FreeBSD 12-STABLE, and MacOS X Monterey, and for every one of them that has a GUI, while some call it something different (Super+L, Meta+L, Windows+L, Command+L), it has the same function on each; it locks the workspace.  Even on Manjaro with KDE Plasma, Super+L locks the workspace.  On the Mac, Command+L locks the workspace (which if you have a Thunderbolt dock with a PC keyboard is the same key).  This is extremely common and standard behavior across every popular OS and most Linux DEs.  This is not Windows behavior.  Though even if it was, Cinnamon tries awfully hard to look and feel like Windows, so why break with it here?

It's baffling that Cinnamon would change this default, let alone make it a keybinding that 99% of the audience would have no reason to ever press.  Honestly I can't even imagine the few people who have cause to use Melange even know or use this keybinding.  Frankly it seems more likely that the person who set this keybinding was ignorant of the very long history and widespread use of Super+L.

It makes far more sense for the one person who apparently uses Melange so much that they need a Super+L keybinding for it to change the default than for every other Cinnamon user who has to frequently switch between environments to have to press a different keybind on Cinnamon (Ctrl+Alt+L) or change it back to the normal default.

**Comment by anatolij-novoselcev** [comment]
You can change this settings

1. https://i.imgur.com/lZMrIAP.png
2. https://i.imgur.com/so7nC2C.png
3. https://i.imgur.com/d15e1Ci.png

**Comment by ciphersimian** [comment]
Yes, I was aware of that, and I have changed this on my system, but that's not what this issue is about.  This issue is about the default in Cinnamon, i.e.

> It makes far more sense for the one person who apparently uses Melange so much that they need a Super+L keybinding for it to change the default than for every other Cinnamon user who has to frequently switch between environments to have to press a different keybind on Cinnamon (Ctrl+Alt+L) **or change it back to the normal default.**

**Comment by Secret-chest** [comment]
I use this pattern for keyboard shortcuts:
* Super: Operating system / desktop (Super-Tab for next workspace, Super-Shift-Tab for previous workspace, Super-L to lock, Super-End to end session, etc.)
* Alt: Window (Alt-Tab / Alt-Shift-Tab to switch, Alt-F4 to close, Alt-* for window menu, Alt-- to minimise)
* Control / shift alone are left for in-app shortcuts / typing.

**Comment by dah33** [comment]
I agree with @ciphersimian. I came over from Windows to Mint as my first Linux desktop, and naturally tried Windows+L (yes it's a Windows themed keyboard) to lock. Melange popped up, definitely not something I'd heard of, or expected.

**Comment by feiyuyitiao** [comment]
https://i.imgur.com/lZMrIAP.png
https://i.imgur.com/so7nC2C.png
https://i.imgur.com/d15e1Ci.png  
i cann't open the link. 😒
